app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
```

Comment:
Nêu ý tưởng code chức năng log in

Reply:
Khi người dùng nhập vào email và password sẽ tạo ra 1 sessions với method là create, tại method create sẽ tìm email của người dùng và sau đó check password, nếu đúng password thì sẽ trở về trang thông tin người dùng, không thì sẽ trở về trang login và đưa ra thông báo.